app/assets/stylesheets/content/_forms.sass

```diff
@@ -244,6 +241,8 @@ fieldset.form--fieldset
 .form--label
   @include grid-content(2)
   @extend %label-style
+  overflow:     visible
+  overflow-y:   visible
```

Comment:
Not sure how precendence stuff works here, but this looks weird.

Isn't `overflow` the same as `overflow-x` **and** `overflow-y`?

I'd either expect a one-liner: `overflow: visible` or when as a two-liner I'd expect:

````CSS
overflow-x: visible;
overflow-y: visible;
````